Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.14 as build-env
+ENV GO111MODULE=on \
+    CGO_ENABLED=0 \
+    GOOS=linux \
+    GOARCH=amd64
+
+WORKDIR /tmp/secrets-store-csi-driver-provider-gcp
+COPY . ./
+RUN go install \
+    -trimpath \
+    -ldflags "-extldflags '-static'" \
```

Comment:
I still think you need all the netgo tags, but maybe it's fine?

Reply:
`CGO_ENABLED=0` prevents using the host dns resolver and will use the netgo one. The build tags are `!cgo netgo` which is why I believe that the `-tag netgo` is not needed. Go 1.15 seems to better document this explicitly and adds more contro: https://golang.org/doc/go1.5#net